tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -1480,7 +1480,7 @@ func getCheckCacheHits(promAPI v1.API, app string) (float64, error) {
 		totalCheck = 0
 	}
 
-	query = fmt.Sprintf("sum(envoy_http_mixer_filter_total_remote_check_calls{app=\"%s\"})", app)
+	query = fmt.Sprintf("sum(envoy_http_mixer_filter_total_remote_check_calls{app.kubernetes.io/name=\"%s\"})", app)
```

Comment:
please revert this part of the changes. The labels used in queries should not be changed.

/cc @gargnupur